code/modules/mob/living/status_procs.dm

```diff
@@ -409,6 +409,8 @@
 		update_blindness()
 
 /mob/living/proc/become_blind(source)
+	if(HAS_TRAIT(src, TRAIT_TRUE_SIGHT))
+		return
```

Comment:
this will cause trouble if the mutation is lost before the source of blindness is, and some, like brain traumas, aren't related to the presence of eyes

Reply:
Yeah, damn that this is kind of a tough nut to crack. I need to figure out what traits people SHOULD have for blindness. Maybe keeping the trait but having it not do anything if true sight is also a trait? I'll think on it

Reply:
i'm starting to really warm up to the idea of keeping all added blind traits but only applying the background if true sight isn't there